test/test_dispatch_onenodestorage.jl

```diff
@@ -0,0 +1,32 @@
+using GENeSYS_MOD
+using HiGHS
+using Ipopt
+
+model, data = genesysmod_dispatch(; solver=HiGHS.Optimizer, DNLPsolver = Ipopt.Optimizer, threads=0, 
+inputdir = joinpath(pkgdir(GENeSYS_MOD),"test","TestData","Inputs"),
+resultdir = joinpath(pkgdir(GENeSYS_MOD),"test","TestData","Results"),
+data_file="RegularParameters_testdata",
+hourly_data_file = "Timeseries_testdata",
+year=2050,
+switch_investLimit=1,
+switch_ccs=1,
+switch_ramping=0,
+switch_weighted_emissions=1,
+switch_intertemporal=0,
+switch_base_year_bounds = 0,
+switch_peaking_capacity = 0,
+set_peaking_slack = 0,
+set_peaking_minrun_share = 0,
+set_peaking_res_cf = 0,
+set_peaking_startyear = 2025,
+switch_peaking_with_storages = 0,
+switch_peaking_with_trade = 0,
+switch_peaking_minrun = 0,
+switch_employment_calculation = 0,
+switch_endogenous_employment = 0,
+employment_data_file = "",
+elmod_dunkelflaute= 0,
+switch_raw_results = CSVResult(),
```

Comment:
When looking for investment results, can we have flexibility in looking for either CSV or TXT? Currently crashes if dispatch looks for TXT and investment run was done printing CSV

Reply:
The switch_raw_results, sets both the format for saving the raw results and the the frmat of results that ot looks for when reading capacities for the dispatch. This means that the dispatch run must use the same format as the dispatch run. This was by design but we could change it. We could introduce a new type TXTandCSV for instance or BOTH or something that would save in both formats.